docs/inference_api_beta/README.md

```diff
@@ -0,0 +1,297 @@
+# Gretel Tabular LLM Inference API (Beta)
+
+This README details the API that powers the [Gretel Model Playground](https://console.gretel.ai/playground).  This API is in **Beta** and subject to change at any time. If you have used the Model Playground and are interested in building your own application with Gretel, then you may uitlize the API directly based on the documentation below. 
+
+Within this directory there are examples with Javascript and Python that provide lighweight wrappers around the API.
+
+# General Operation
+
+Gretel Tabular LLM utilizes lightweight streams to generate and retrieve tabular data. Compared to other LLM streaming mechanisms, such as Server-Sent Events (SSE), Gretel does not stream back individual tokens. The Tabular LLM infrastructure automatically utilizes multiple LLMs and Agents to generate tokens and re-assemble them into tabular data. Client-side applications can then poll specific REST API endpoints to retrieve table rows as they are generated and assembled. By using a lightweight polling mechanism, Tabular LLM can be more flexibly integrated into apps that may not have streaming capabilities.
+
+There are two main steps to utilize the Tabular LLM APIs:
+
+1) First, a request is made that contains the prompt and LLM parameters. This request will return immediately with a *inference stream ID*. This ID represents a lightweight stream that can be itereated next. Behind the scenes, this request will be processed and tabular records will be generated and written to the stream.
```

Comment:
```suggestion
1) First, a request is made that contains the prompt and LLM parameters. This request will return immediately with a *inference stream ID*. This ID represents a lightweight stream that can be iterated through. Behind the scenes, this request will be processed and tabular records will be generated and written to the stream.
```

Reply:
its still ending a sentence w/ a preposition so its not a much better fix, feel free to rephrase